module/platform/Webots/src/Webots.cpp

```diff
@@ -1,21 +1,53 @@
-#include "webots.hpp"
-
-#include <fmt/format>
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2021 NUbots <nubots@nubots.net>
+ */
+
+#include "Webots.hpp"
+
+#include <fmt/format.h>
 
 #include "extension/Configuration.hpp"
 
-#include "message/motion/ServoTarget.proto"
-#include "message/output/CompressedImage.proto"
-#include "message/platform/darwin/DarwinSensors.proto"
-#include "message/platform/webots/ConnectRequest.proto"
-#include "message/platform/webots/messages.proto"
+#include "message/motion/ServoTarget.hpp"
+#include "message/output/CompressedImage.hpp"
+#include "message/platform/darwin/DarwinSensors.hpp"
+#include "message/platform/webots/ConnectRequest.hpp"
+#include "message/platform/webots/messages.hpp"
+
+// Include headers needed for TCP connection
+#ifdef _WIN32
+    #include <winsock.h>
+#else
+    #include <netdb.h>      /* definition of gethostbyname */
+    #include <netinet/in.h> /* definition of struct sockaddr_in */
+    #include <sys/socket.h>
+    #include <sys/time.h>
+    #include <unistd.h> /* definition of close */
+#endif
```

Comment:
Do we need to include Windows headers as we always run on the docker image?

Reply:
I don't think so?

Reply:
From the slack discussion, "Probably not".